autofs-5.1.0 - make service want network-online

From: Ian Kent <ikent@redhat.com>

autofs often fails to start properly in Fedora with recent systemd.

Changing the systemd unit to Want the network-online target works
around this.

I'm not sure if this will cause problems for people that use file
maps and expect autofs to start without the network up but I hope
that's a small minority, if there are any at all.
---
 CHANGELOG                 |    1 +
 samples/autofs.service.in |    3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -182,6 +182,7 @@
 - fix direct map expire not set for initail empty map.
 - update map_hash_table_size description.
 - fix out of order call in program map lookup.
+- make service want network-online.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/samples/autofs.service.in
+++ autofs-5.0.7/samples/autofs.service.in
@@ -1,6 +1,7 @@
 [Unit]
 Description=Automounts filesystems on demand
-After=network.target ypbind.service sssd.service
+After=network.target ypbind.service sssd.service network-online.target
+Wants=network-online.target
 
 [Service]
 Type=forking
